### DISCUSSION TITLE
how to running the flow in fix interval?

### DISCUSSION THREAD

**Discussion by imknight** [directus/directus#16816] [Archive]
For example, I want to re-send notifications to users every 3 days since they created a record, any idea how can I do this?

**Comment by alexvdvalk** [comment]
I believe the best way to achieve this is to use a CRON flow. You can set this to run every 3 days, and then get the relevant records from the database with a query.

I'm no cron expert, so check out https://crontab.guru/#0_0_*/3_*_* to view and build cron expressions.

**Reply by paescuj** [reply]
Thanks @alexvdvalk 👍 Crontab.guru is a great resource 😃 It seems like GitHub fails to correctly display the plain link (missing wildcards), so here's the correct link: [`0 0 */3 * *`](https://crontab.guru/#0_0_*/3_*_*)

**Reply by imknight** [reply]
This is the first way I m trying but in this way, it won't be accurate in 3 days interval because the corn job only runs in 3 days interval.
Is it possible to run the cron daily and checking on the created_date modulus with 3 ? at least that's how I normally do in PHP.

**Reply by paescuj** [reply]
I'm afraid I don't quite understand. Do you want to run the job on fixed days (e.g. something like `0 0 1,4,7 * *`) or related to some other value?
Anyway, you can use the "Schedule (CRON)" trigger set to a daily basis, then connect it to a "[Run Script](https://docs.directus.io/configuration/flows/operations.html#run-script)" operator to do further checks.

**Reply by imknight** [reply]
i m trying to send the notification 3 days interval after the record is created, so if let say the record created on 2nd of Dec, then the notification should send on 5th.

**Reply by paescuj** [reply]
Ah, thanks, I see 👍 
In that case you should be good to go with this:
> Anyway, you can use the "Schedule (CRON)" trigger set to a daily basis, then connect it to a "[Run Script](https://docs.directus.io/configuration/flows/operations.html#run-script)" operator to do further checks.
